packages/forma-36-react-components/src/components/Pill/Pill.stories.tsx

```diff
@@ -4,53 +4,77 @@ import { action } from '@storybook/addon-actions';
 import { text } from '@storybook/addon-knobs';
 import Icon from '../Icon';
 
-import Pill from './Pill';
-
-storiesOf('Components/Pill', module)
-  .addParameters({
-    propTypes: Pill['__docgenInfo'],
-    component: Pill,
-  })
-  .add('default', () => (
+import Pill, {PillProps} from './Pill';
+import notes from './README.md';
+
+
+export default {
+  title: 'Components/Pill',
+  component: Pill,
+  parameters: {
+    propTypes: [Pill['__docgenInfo']],
+    notes,
+  },
+  argTypes: {
+    label: { control: { type: 'text'} },
+    className: { control: { disable: true } },
+    testId: { control: { disable: true } },
+    onClose:{ control: { disable: true } },
+    onDrag:{ control: { disable: true } },
+    dragHandleComponent: { control: { disable: true } },
+  },
+};
+
+export const basic = (args: PillProps) => (
+  <Pill
+      label={args.label}
+    />
+);
+
+export const onDragAndOnClose = (args: PillProps) => (
+  <>
     <div>
       <Pill
-        className={text('className', '')}
-        label={text('label', 'johannes.bugiel@contentful.com')}
+        label={args.label}
+        onClose={args.onClose}
+        onDrag={args.onDrag}
       />
     </div>
-  ))
-  .add('onDrag & onClose', () => (
-    <>
-      <div>
-        <Pill
-          className={text('className', '')}
-          label={text('label', 'johannes.bugiel@contentful.com')}
-          onClose={action('onClick')}
-          onDrag={action('onDrag')}
-        />
-      </div>
-      <div className="f36-margin-top--m">
-        <Pill
-          style={{ width: 200 }}
-          className={text('className', '')}
-          label={text('label', 'johannes.bugiel@contentful.com')}
-          onClose={action('onClick')}
-          onDrag={action('onDrag')}
-        />
-      </div>
-    </>
-  ))
-  .add('Custom handle component', () => (
-    <Pill
-      className={text('className', '')}
-      label={text('label', 'johannes.bugiel@contentful.com')}
-      onDrag={action('onDrag')}
-      dragHandleComponent={
-        <Icon
-          icon="ArrowUp"
-          color="positive"
-          style={{ padding: '0.375rem 0.625rem' }}
-        />
-      }
-    />
-  ));
+    <div className="f36-margin-top--m">
+      <Pill
+        style={{ width: 200 }}
+        label={args.label}
+        onClose={args.onClose}
+        onDrag={args.onDrag}
+      />
+    </div>
+  </>
+);
+
+export const CustomHandleComponent = (args: PillProps) => (
+  <Pill
+    label={args.label}
+    onDrag={args.onDrag}
+    dragHandleComponent={
+      <Icon
+        icon="ThumbUp"
+        color="primary"
+        style={{ padding: '0.375rem 0.625rem' }}
+      />
+    }
+  />
+);
+
+basic.args = { label: 'mike.mitchell@contentful.com' };
+
+onDragAndOnClose.args = {
+  label: 'guilherme.santos@contentful.com ',
+  onClose: action('clicked'),
+  onDrag: action('dragged'),
+};
+
+CustomHandleComponent.args = {
+  label: 'moe.shaaban@contentful.com',
```

Comment:
Should we replace these with dummy emails? GDPR 🚔

Reply:
@mshaaban0 hehe, yeah, this is what I thought. It was Johannes email before. I will change it to something else. on my way 🚀